python/llm/example/CPU/HF-Transformers-AutoModels/Model/glm4/README.md

```diff
@@ -17,7 +17,7 @@ conda activate llm
 # install the latest ipex-llm nightly build with 'all' option
 pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu
 
-# install packages required for GLM-4
+# install packages required for GLM-4, it is recommended to use transformers>=4.44 for glm-4-9b-chat updated after August 12, 2024
```

Comment:
It is recommended to use transformers>=4.44 for THUDM/glm-4-9b-chat updated after August 12, 2024